src/color-picker/color-picker.scss

```diff
@@ -250,17 +229,36 @@ $iui-hover-box-shadow: 0 0 1px $iui-xxs + 1 rgba(t(iui-color-foreground-body-rgb
   border-radius: 50%;
   transform: translate(-$iui-s, -$iui-s);
   cursor: crosshair;
-  @include themed {
```

Comment:
Maybe put `themed` mixin back, or change `t(` to `var(--`

Reply:
Ahh, I did miss a couple.  I've swapped the `t(` that are no longer within `themed` with `var(--`.